backend/queues/txProcessor.js

```diff
@@ -68,9 +68,11 @@ async function processor(job) {
 
   // Load the transaction from the DB.
   const transaction = await Transaction.findOne({
-    networkId: network.networkId,
-    shopId,
-    hash: txHash
+    where: {
```

Comment:
wow. thanks for catching this!!! interesting that the unit test did not catch it... I guess during testing sqlite must have alway been returning the latest row in the transaction table...